jupyter-extension/jupyterlab_pachyderm/tests/test_handlers.py

```diff
@@ -603,96 +599,3 @@ async def test_get_projects(mock_client, jp_fetch):
     mock_client.list_projects.return_value = json.dumps(test_projects)
     resp = await jp_fetch(f"/{NAMESPACE}/{VERSION}/projects")
     assert json.loads(resp.body) == test_projects
-
```

Comment:
Would these tests be useful to keep for when we add "write config details to config file"?

Reply:
i figure we can add a similar test back when we implement that functionality; this tests the now obsolete `write_token_to_config` method which takes the session token from mount-server and writes it into the config. i think we will want to write a more general purpose config handler (either here, or in the pachyderm_sdk) and add general config file testing when we implement that.

Reply:
i figure we can add a similar test back when we implement that functionality; this tests the now obsolete `write_token_to_config` method which takes the session token from mount-server and writes it into the config. i think we will want to write a more general purpose config handler (either here, or in the pachyderm_sdk) and add general config file testing when we implement that.